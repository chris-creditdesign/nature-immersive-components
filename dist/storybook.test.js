"use strict";

var _addonStoryshots = _interopRequireDefault(require("@storybook/addon-storyshots"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

(0, _addonStoryshots.default)();
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uL3NyYy9zdG9yeWJvb2sudGVzdC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiOztBQUFBOzs7O0FBRUEiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgaW5pdFN0b3J5c2hvdHMgZnJvbSBcIkBzdG9yeWJvb2svYWRkb24tc3RvcnlzaG90c1wiXG5cbmluaXRTdG9yeXNob3RzKClcbiJdfQ==