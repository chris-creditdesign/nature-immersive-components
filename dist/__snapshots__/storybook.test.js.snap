// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Test default 1`] = `
<p>
  Test component
</p>
`;
